src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -120,18 +132,19 @@ private void bindLauncherControls() {
 						s.launcherSubsystem,
 						new SetAngleCommand(
 								s.launcherSubsystem,
-								() -> MathUtil.applyDeadband(codriveController.getLeftY(), 0.1) * 0.75));
+								() -> MathUtil.applyDeadband(codriveController.getLeftY(), 0.1) * 0.1));
```

Comment:
Note that this may be too low, as the max speed won't exceed 0.1. Should be okay to leave as-is though just to be safe.

Reply:
Yeah, i just wanted to put something i knew would be safe and wouldnt like destroy everything lol. ill turn it up to .3 tho